docs/install.md

```diff
@@ -20,19 +37,12 @@ brew install brimdata/tap/super
 
 Once installed, run a [quick test](#quick-tests).
 
-## Binary Download
-
-We offer pre-built binaries for macOS, Windows and Linux for both amd64/x86 and arm
-architectures in the super [GitHub Release page](https://github.com/brimdata/super/releases).
-
-Once the `super` binary is unpacked from a downloaded package, run a [quick test](#quick-tests).
-
```

Comment:
Since the plan is to hide the Releases link from the repo's front page to reflect that there's not yet GA SuperDB releases, it seems we should drop it here as well. Legacy Zed users will hopefully find their way to the older releases via https://www.brimdata.io and https://www.brimdata.io/download .